alcs-frontend/src/app/features/application/overview/overview.component.ts

```diff
@@ -89,6 +89,14 @@ export class OverviewComponent implements OnInit {
       });
     }
 
+    if (application.notificationSentDate) {
+      mappedEvents.push({
+        name: "'Ready for Review' Notification Sent to Applicant",
```

Comment:
Do we need to adjust the casing in name?

Reply:
Everything else in the timeline is currently title cased, so this follows that pattern